Src/Amr/AMReX_extrapolater_2D_K.H

```diff
@@ -0,0 +1,143 @@
+#ifndef AMReX_extrapolater_2D_K_H_
+#define AMReX_extrapolater_2D_K_H_
+
+namespace amrex {
+
+AMREX_GPU_HOST_DEVICE
+AMREX_FORCE_INLINE
+void
+amrex_first_order_extrap(amrex::Box const& bx,
+                         int               nComp,
+                         amrex::Array4<const int>   const& mask,
+                         amrex::Array4<amrex::Real> const& data) noexcept
+{
+   using namespace amrex::literals;
+
+   const int finecell = 1;
+   const int crsecell = 0;
+
+   const auto lo = amrex::lbound(bx);
+   const auto hi = amrex::ubound(bx);
+   int idlo[3] = {D_DECL(lo.x,lo.y,lo.z)};            // base box bounds
+   int idhi[3] = {D_DECL(hi.x,hi.y,hi.z)};
+   int idlo_e[3] = {D_DECL(lo.x,lo.y,lo.z)};          // extended box bounds
+   int idhi_e[3] = {D_DECL(hi.x,hi.y,hi.z)};
```

Comment:
Have to use `AMREX_D_DECL`.

Reply:
Indeed. I'll have to go over PeleLM to change that too ...

Reply:
Hmm, maybe should declare as int[AMREX_SPACEDIM], just to be more explicit?

Reply:
At some point I had 1D/2D and 3D in the same file but Kevin suggested splitting the files to be more compliant with AMReX style and it's a remnant of that. I'll actually clean that up.